volume/volume.go

```diff
@@ -104,6 +104,8 @@ type SnapshotDriver interface {
 	Snapshot(volumeID string, readonly bool, locator *api.VolumeLocator) (string, error)
 	// Restore restores volume to specified snapshot.
 	Restore(volumeID string, snapshotID string) error
+	// GroupSnapshot takes a snapshot of specified volumegroup.
+	SnapshotGroup(groupID string, labels map[string]string) (*api.GroupSnapCreateResponse, error)
```

Comment:
Comment says GroupSnapshot but API is SnapshotGroup and the response is GroupSnap. Make it consistent.
Also better to define GroupSnapCreateRequest to pass in parameters IMO.